simplex-solver/README.md

```diff
@@ -0,0 +1,33 @@
+# cljsjs/simplex-solver
+
+[](dependency)
+```clojure
+[cljsjs/simplex-solver "0.0.3-0"] ;; latest release
+```
+[](/dependency)
+
+This jar comes with `deps.cljs` as used by the [Foreign Libs][flibs] feature
+of the Clojurescript compiler. After adding the above dependency to your project
+you can require the packaged library like so:
+
+```clojure
+(ns application.core
+  (:require cljsjs.simplex-solver))
+```
+
+[flibs]: https://github.com/clojure/clojurescript/wiki/Packaging-Foreign-Dependencies
+
+## Usage
+
+If you include the `cljs.simplex-solver` namespace anywhere in your
+ClojureScript code, simplex-solver will be available as the Javascript
+global `js/simplex`.
+
+
+```clojure
+(.maximize js/simplex "x + y + 2z" (clj->js ["2x + y + z <= 50"
```

Comment:
As minor note, `#js` is better than `clj->js` here. `clj->js` does the transformation in runtime, but `#js` will tell the compiler to emit JS Array instead of Clojure vector.